testassets/InteropTestsClient/RunGrpcWebTests.ps1

```diff
@@ -0,0 +1,28 @@
+﻿Param
+(
+    [bool]$use_tls = $false
+)
+
+$allTests =
+  "grpcweb_unary",
+  "grpcweb_server_streaming",
+  "grpcweb_unary_abort",
+  "grpcweb_server_streaming_abort"
+
+Write-Host "Running $($allTests.Count) tests" -ForegroundColor Cyan
+Write-Host "Use TLS: $use_tls" -ForegroundColor Cyan
+Write-Host
+
+foreach ($test in $allTests)
+{
+  foreach ($grpcWebMode in "GrpcWeb","GrpcWebText")
+  {
+    Write-Host "Running $grpcWebMode $test" -ForegroundColor Cyan
+
+    dotnet run --use_tls $use_tls --server_port 8080 --client_type httpclient --grpc_web_mode $grpcWebMode --test_case $test
```

Comment:
I'm assuming that with the right infrastructure in place, these tests can run against the code in grpc-web, but we are not currently testing that?

Reply:
No. https://github.com/johanbrandhorst/grpc-web-compatibility-test has Docker containers that allow you to manually run implementations against one another, but I couldn't see anything that is automated